### PR TITLE
Feature/display values 3d model

### PR DESCRIPTION
**Pull request by JakobRathberger**

### CODE REVIEW

File: src/app/core/core.module.ts [hunk 1]
```diff
@@ -18,11 +18,11 @@ import {
...
+  // path: `${environment.mqttPathPrefix}${environment.mqttPath}`,
+  protocol: 'ws',
   port: environment.mqttPort,
-  username: 'ui_client',
-  password: 'ud84jc23'
+  username: 'student',
+  password: 'passme'
 };
 
 @NgModule({
```
Comment (quirinecker) [inline src/app/core/core.module.ts:28]: you should maybe extract the user credentials into secrets.

File: src/environments/environment.prod.ts [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
-  corePathPrefix: '/core',
+  vmUrl: 'https://vm139.htl-leonding.ac.at',
+  mqttUrl: 'vm90.htl-leonding.ac.at',
+  corePathPrefix: '',
   mqttPathPrefix: '/broker',
   mqttPath: '/mqtt',
-  mqttPort: 443
+  mqttPort: 9001,
+  vmPort: 8080
 };
```
Comment (quirinecker) [inline src/environments/environment.prod.ts:10]: the url for the production environment will probably be leo-iot-server if i am not mistaken

**Review by quirinecker — COMMENTED** [review]
everything else should be fine

File: src/app/core/core.module.ts (hunk 1 above)
Comment (JakobRathberger) [reply src/app/core/core.module.ts:28]: extracting the user credentials is planned for a future PR

Comment (quirinecker) [reply src/app/core/core.module.ts:28]: Kk

**Review by philippkerschbaum — APPROVED** [review]

**Pull request merged**